ui/src/index.css

```
@@ -2487,22 +2238,77 @@ tr:hover td {
+  font-size: 1.15rem;
+  font-weight: 700;
+  color: var(--mj-gold);
+  background: linear-gradient(135deg, var(--mj-gold), #f0c040, var(--mj-gold));
+  background-size: 200% 200%;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+  animation: heroGradient 4s ease infinite;
+}
+
+@keyframes heroRingPulse {
```

Stylelint (17.9.0)

[error] Expected keyframe name "heroRingPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)